### PR TITLE
Add docs for shell aliasing & clarifications

### PR DESCRIPTION
**Pull request by markstory**
Add docs for shell aliasing and try to clarify some of the gaps pointed out in #2847.

### CODE REVIEW

File: en/tutorials-and-examples/bookmarks/intro.rst [hunk 1]
```diff
@@ -336,26 +345,31 @@ following content::
...
     <?php endforeach; ?>
     </section>
 
-CakePHP expects that our templates follow the naming convention where the
-template has the lower case and underscored version of the controller action
-name.
+In the above code we use the :doc:`/views/helpers/html` and
+:doc:`/views/helpers/text` helpers to assist in generating our view output. We
+also use the :php:func:`h` shortcut function to HTML encode output. You should
+remember to always use ``h()`` when outputting user data to prevent HTML
```
Comment (jadb) [inline en/tutorials-and-examples/bookmarks/intro.rst]: Should this be displayed in an alert box or something?

**Comment by jadb** [comment]
:+1:

File: en/tutorials-and-examples/bookmarks/intro.rst (hunk 1 above)
Comment (markstory) [inline en/tutorials-and-examples/bookmarks/intro.rst]: Could be.

File: fr/tutorials-and-examples/bookmarks/intro.rst [hunk 2]
```diff
@@ -356,15 +365,26 @@ Construisons donc le fichier de vue pour notre action ``tags()``. Dans
...
+
+            <!-- Utilise le TextHelper pour formater le texte -->
             <?= $this->Text->autoParagraph($bookmark->description) ?>
         </article>
     <?php endforeach; ?>
     </section>
 
-CakePHP s'attend à ce que nos templates suivent la convention de nommage où
-le template a la version en minuscule et en underscore du nom de l'action du
+Dans le code ci-dessus, nous utilisons le :doc:`Helper HTMl </views/helpers/html>`
```
Comment (HavokInspiration) [inline fr/tutorials-and-examples/bookmarks/intro.rst]: Missing a capital L to "HTML"

File: fr/tutorials-and-examples/bookmarks/intro.rst [hunk 3]
```diff
@@ -336,7 +341,11 @@ Dans **src/Model/Table/BookmarksTable.php** ajoutez ce qui suit::
 
 Nous intégrons juste :ref:`des finders personnalisés <custom-find-methods>`.
 C'est un concept très puissant dans CakePHP qui vous permet de faire un package
-réutilisable de vos requêtes. Dans notre finder nous avons amené la méthode
+réutilisable de vos requêtes. les finders attendent toujours un objet
```
Comment (HavokInspiration) [inline fr/tutorials-and-examples/bookmarks/intro.rst]: Missing a capital L to "Les"

File: fr/tutorials-and-examples/bookmarks/intro.rst [hunk 4]
```diff
@@ -336,7 +341,11 @@ Dans **src/Model/Table/BookmarksTable.php** ajoutez ce qui suit::
...
+:doc:`/orm/query-builder` et un tableau d'options en paramètre. les finders
```
Comment (HavokInspiration) [inline fr/tutorials-and-examples/bookmarks/intro.rst]: Missing a capital L to "Les"

File: fr/tutorials-and-examples/bookmarks/intro.rst [hunk 5]
```diff
@@ -356,15 +365,26 @@ Construisons donc le fichier de vue pour notre action ``tags()``. Dans
...
-le template a la version en minuscule et en underscore du nom de l'action du
+Dans le code ci-dessus, nous utilisons le :doc:`Helper HTMl </views/helpers/html>`
+et le :doc:`Helper Text </views/helpers/text>` pour aider à la génération
+du contenu de notre vue. Nous utilisons également la fonction :php:func:`h`
+pour encoder la sortie en HTML. Vous devez vous rappeler de toujours utiliser
+``h()`` lorsque vous affichez des données provenant des utilisateurs pour éviter
+les problèmes d'injection HTML.
+
+Le fichier ``tags.ctp`` que nous venons de créer suit la convention de nommage
+de CakePHP pour un ficher de template de vue. la convention d'avoir le nom
```
Comment (HavokInspiration) [inline fr/tutorials-and-examples/bookmarks/intro.rst]: Missing a capital L to "La"

File: fr/tutorials-and-examples/bookmarks/intro.rst [hunk 6]
```diff
@@ -356,15 +365,26 @@ Construisons donc le fichier de vue pour notre action ``tags()``. Dans
...
+de template en version en minuscule et en underscore du nom de l'action du
```
Comment (HavokInspiration) [inline fr/tutorials-and-examples/bookmarks/intro.rst]: ~~en version~~ en minuscule

File: fr/console-and-shells.rst [hunk 7]
```diff
@@ -89,6 +89,23 @@ La première information affichée est en rapport avec les chemins. Ceci est
...
+    # lance le shell server
+    bin/cake server
+
+    # lance le shell migrations
+    bin/cake migrations -h
+
+    # lance le shell bake (avec le préfie plugin)
+    bin/cake bake.bake -h
+
+Les shells de plugins peuvent être invoqué sans le préfixe du plugin si le nom
```
Comment (HavokInspiration) [inline fr/console-and-shells.rst]: invoqués

File: fr/console-and-shells.rst [hunk 8]
```diff
@@ -89,6 +89,23 @@ La première information affichée est en rapport avec les chemins. Ceci est
...
+du shell n'entre pas en collision avec un shell de l'application ou du
+framework. Dans le cas où deux plugins fournissent un shell du même nom, c'est
+le le premier chargé qui récupérera l'alias court. Vous pouvez toujours
```
Comment (HavokInspiration) [inline fr/console-and-shells.rst]: Double "le"

**Comment by antograssiot** [comment]
@HavokInspiration Merci pour la relecture Yves, y a des jours où y a pas moyen :sleepy:

**Pull request merged**